terra-batch-libchart/values.yaml

```diff
@@ -4,6 +4,7 @@ exports:
       name: coa-cromwell-svc
       image: broadinstitute/cromwell:85-46a7918
       port: 8000
+      coaEnabled: false
```

Comment:
If think we should start with these enabled by default, and allow Leo to override and disable these apps. This way, we don't impact CBAS & cromwell apps testing in dev. This won't address the WDS challenges testing in dev

Reply:
Are you saying like referencing a `RUNTIME_PARAMETER` instead?

Reply:
Well, Leo can always override whatever is in `values.yaml`. Ideally though I think we'd like the dev config to reflect prod so we can test it, I think. Maybe we could have WDS-only in dev, with the ability to add other services via config change in BEEs?

Reply:
Needs a bit more testing, but opened this Leo PR to control which coa services are enabled: https://github.com/DataBiosphere/leonardo/pull/3128

Leo also needs to know which services to status-check, so it makes sense for Leo to control it, I think. This way we could also enable the other ones with a Leo config change, instead of a new chart version.

Reply:
OK after discussion, I think we'll keep enabling all the services in dev, and use a BEE to test the WDS-only configuraiton. Will update Leo PR.

Reply:
Maybe default these all to true, given that Leo will be environmentally controlling it